File: daily_challenges/976.py
def largestPerimeter(nums):
    n = len(nums)
    nums.sort()
    for i in range(n-3, -1, -1):
        if(nums[i] + nums[i+1] > nums[i+2]):
            return nums[i] + nums[i+1] + nums[i+2]
    return 0

File: daily_challenge_leetcode/976.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var largestPerimeter = function (nums) {
    const n = nums.length
    nums.sort((a, b) => a - b)
    for (let i = n - 3; i >= 0; i--) {
        if (nums[i] + nums[i + 1] > nums[i + 2]) {
            return nums[i] + nums[i + 1] + nums[i + 2]
        }
    }
    return 0
};